DeAceroDLabs/marketplace-frontend
# fix: [DMP-482] Loads default image on card if not image found.

## What is done

[[DMP-482] default image on card if not image found](https://deacero.atlassian.net/browse/DMP-482)

## How is done

- Added to `Card` component a broken image icon that shows when no imgScr is given.
- Added `product.categoryImageUrl ?? ""`  in  `Search` component to check if no imgScr is given.

## How to test

1. Run the project via `npm start`.
2. login.
3. Search for clavo in the search bar.
4. You should seethe broken image icon.
